articles/tokens/access_token-vs-id_token.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Auth0 access_token vs id_token
+---
+
+# Auth0 access_token vs id_token
+
+## When should you use the `access_token` and the `id_token`?
+
+One of the common questions that gets asked is when you should use the `access_token` and when you should use the `id_token`? 
+
+The answer is as follows:
+
+* If you need to pass a user's credentials to your own API in order to authenticate the user against the API, then you should use the `access_token`
+* When you want to get information about a user from the [/tokeninfo endpoint](/auth-api#post--tokeninfo), then you should use the `id_token`
```

Comment:
Yes.  The `id_token` should never be supplied to any API, including `/tokeninfo` and `/userinfo`.

There are some cases involving delegation (true delegation, not our version of it) that may use an id_token.  We are a long way off from implementing those, so I don't think those are a concern for now.

I would rephrase this as:

- The access token should be sent as credentials when making API calls on behalf of a user.
- The id_token is used by the client to obtain profile information about the user, by decoding and verifying the claims in the JWT.  The id_token should never be sent to an API.  The id_token is an optimization to avoid network requests to the /userinfo endpoint.   If the /userinfo API is called using the access_token, it should return the same claims that are encoded directly in the id_token.   Note that in some cases, the /userinfo endpoint may return *more* information, due to size constraints with JWT id_tokens.

Something like that...